tests/testTriggers.go

```diff
@@ -5537,6 +5539,20 @@ func TriggerMetadataPoolResizeDisk(contexts *[]*scheduler.Context, recordChan *c
 		wg.Wait()
 
 	})
+
+	stNodes := node.GetStorageNodes()
+	expectedCapacity := (len(stNodes) / 2) + 1
+	poolsToExpand := make([]*opsapi.StoragePool, 0)
+	for _, stNode := range stNodes {
+		eligibility, err := GetPoolExpansionEligibility(&stNode)
+		log.InfoD(fmt.Sprintf("error getting the pool expansion eligibility :%s", err))
+		if len(poolsToExpand) <= expectedCapacity {
```

Comment:
Might be unnecessary.

Reply:
removed the code and added new one to retirve node from a pool uuid